src/Console/Command.php

```diff
@@ -102,6 +103,25 @@ public function getName()
         return $this->name;
     }
 
+    /**
+     * Get the command name.
+     *
+     * Returns the command name based on class name.
+     * For e.g. for a command with class name `UpdateTableCommand` the default
+     * name returned would be `'update_table'`.
+     *
+     * @return string
+     */
+    public static function defaultName()
+    {
+        $pos = strrpos(static::class, '\\');
+        /** @psalm-suppress PossiblyFalseOperand */
```

Comment:
I dont think we use the `psalm` in 3.x

Reply:
but wouldnt be a big deal to keep, either.

Reply:
yes